css/src/components/toggle.scss

```diff
@@ -0,0 +1,86 @@
+$toggle-transition: $input-transition-duration cubic-bezier(0.01, 1.23, 0.58, 0.96) !default;
+$toggle-circle-size: 0.75em !default;
+$toggle-circle-top: 0.1875em !default;
+$toggle-circle-distance-from-border: 1em - $toggle-circle-size;
+
+$toggle-unselected-circle-color: $text-subtle !default;
+$toggle-unselected-border-color: $text-subtle !default;
+$toggle-unselected-background-color: $body-background !default;
+$toggle-selected-background-color: $primary !default;
+$toggle-circle-selected-background-color: $body-background !default;
+
+.toggle {
+	label {
+		display: block;
+		position: relative;
+		width: 2.5em;
+		height: 1.25em;
+		transition: $input-transition-duration linear;
+		transition-property: background-color, border-color;
+		border: 0.065em solid $toggle-unselected-border-color;
+		border-radius: $radius-rounded;
+		background-color: $toggle-unselected-background-color;
+		cursor: pointer;
+
+		@include forced-colors {
+			border-color: CanvasText !important;
+			background-color: Canvas !important;
+		}
+
+		&::after {
+			display: block;
+			position: absolute;
+			width: $toggle-circle-size;
+			height: $toggle-circle-size;
+			transition: $toggle-transition;
+			transition-property: inset-inline-start, border-color, background-color;
+			border-radius: $radius-rounded;
+			inset-block-start: $toggle-circle-top;
+			inset-inline-start: $toggle-circle-distance-from-border;
+			background-color: $toggle-unselected-circle-color;
+			content: '';
+
+			@include forced-colors {
+				background-color: CanvasText !important;
+			}
+		}
+	}
+
+	/* stylelint-disable -- want to keep [type="checkbox"] as a requirement here  */
+	&.is-selected label,
+	input[type='checkbox']:checked + label {
+		/* stylelint-enable */
+		border-color: $toggle-selected-background-color;
+		background-color: $toggle-selected-background-color;
+
+		@include forced-colors {
+			border-color: LinkText !important;
+			background-color: LinkText !important;
+		}
```

Comment:
I might be wrong, but I think the more "semantic" system colors to use here would be the `Button` ones. Per https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#system_colors,

> **ButtonBorder**
> Base border color of controls
>
> **ButtonFace**
> Background color of controls
>
> **ButtonText**
> Foreground color of controls

The generic "controls" phrasing makes me think that's what we should be using for this. Thoughts?

Reply:
Could be that's right, was thinking that I like that the enabled version of the toggle to be more actively colored. It looks liek this if we use ButtonText. 

![image](https://user-images.githubusercontent.com/30843002/153060014-c6c4d0aa-8e5a-4e43-8fb6-8974d242d6b2.png)

Reply:
Compared to current in PR: 
![image](https://user-images.githubusercontent.com/30843002/153060204-562f700a-c380-4ff3-8594-b815c7d0a164.png)

Reply:
What about using `ButtonFace`? It's for "Background color of controls," so that might be more colorful.

Reply:
Button Face is black in this particular case.

Reply:
Gotcha - then I'm good with how we have it!